### PR TITLE
[test/sensor] Add more options and README.md for developers

### PR DESCRIPTION
**Pull request by again4you**
This patch updates the testcase of nnstreamer-edge-sensor to test various case. Moreover, it newly adds the `REAEME.md` file for developers to build and test nnstreamer-edge-sensor library.

### Patch set v1
* Add README.md for developers
* Add more options to test various case

### CODE REVIEW

File: src/libsensor/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Before executing the server script, you have to check the `mqttsrc` and `mqttsink` are available as below.
+
+```bash
+$ gst-inspect-1.0 --gst-plugin-path="build" | grep mqtt
+mqtt:  mqttsink: MQTT sink
+mqtt:  mqttsrc: MQTT source
+```
+
+If `mqtt` emelemt is not available in your environment, you can build the NNStreaemr from the source code after enabling the mqtt-support option.
```
Comment (gichan-jang) [inline src/libsensor/README.md:62]:
```suggestion
If `mqtt` emelemt is not available in your environment, you can build the NNStreamer from the source code after enabling the mqtt-support option.
```

File: src/libsensor/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# nnstreamer-edge-sensor
+
+`nnstreamer-edge-sensor` is a utility library for publishing any type of data as an MQTT message.
+Without GStreamer dependencies, it can provide remote nodes with stream input data such as camera image, temperature, or humidity. 
+`mqttsrc` element of NNStreaer can access these data and exploit them on pipeline stream.
```
Comment (jaeyun-jung) [inline src/libsensor/README.md:5]: NNStreaer > NNStreamer

File: src/libsensor/README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+`nnstreamer-edge-sensor` is designed not to depend on GStreamer/GLib so targets to be adopted by general RTOS or Lightweight devices.
+
+
+## Getting Started
+
+### Prerequisites
+
+To build `nnstreamer-edge-sensor` in the Ubuntu/Debian envrionment, the following packages are needed.
```
Comment (jaeyun-jung) [inline src/libsensor/README.md:14]: envrionment > environment

File: src/libsensor/README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+```bash
+$ sudo apt install cmake libpaho-mqtt-dev libpaho-mqtt1.3 mosquitto mosquitto-clients
+```
+
+### How to build
+
+When building `nnstreamer-edge-sensor`, you can set the soversion of nnstreamer-edge-sensor or enable test program.
```
Comment (jaeyun-jung) [inline src/libsensor/README.md:22]: soversion > version ?

File: src/libsensor/README.md (hunk 1 above)
Comment (jaeyun-jung) [reply src/libsensor/README.md:62]: `mqtt` emelemt > `mqtt` element

File: src/libsensor/README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+```bash
+$ git clone https://github.com/nnstreamer/nnstreamer.git
+$ cd nnstreamer
+
+# Enable the mqtt-support option as below
+# meson_options.txt
+# +option('mqtt-support', type: 'feature', value: 'enabled')
+
+$ meson build
+$ ninja -C build
```
Comment (jaeyun-jung) [inline src/libsensor/README.md:73]: need install?
`ninja -C build install`

Comment (jaeyun-jung) [reply src/libsensor/README.md:73]: Oh...example with gst-plugin-path option, plz ignore this.

Comment (again4you) [reply src/libsensor/README.md:73]: ACK! :)

**Review by jaeyun-jung — APPROVED** [review]

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Pull request merged**